### DISCUSSION TITLE
low trust via vac bypass

### DISCUSSION THREAD

**Discussion by fanat1cc** [ByMynix/CouInjector#2] [General]
idk if the vac bypass is giving low trust but i have been legit cheating with Osiris (built myself) and have started getting into games with obv cheaters it could be my recent high kill comp games but if you guy's could tell me if it is or is not the vac bypass it would be nice.

**Comment by ByMynix** [comment]
> idk if the vac bypass is giving low trust but i have been legit cheating with Osiris (built myself) and have started getting into games with obv cheaters it could be my recent high kill comp games but if you guy's could tell me if it is or is not the vac bypass it would be nice.

There is a lot of discussion about the VAC-ByPass and whether it causes a lower trust. In short, no one is quite sure, it is possible because an error is sent to VAC during the initialisation of the disabled VAC module. However, I cannot explicitly answer whether it is your higher conspicuousness during the game that makes the trust factor lower or the VAC-ByPass. I myself did a rough test and didn't notice any effects on the trust factor, but as I already said, there are many discussions about this.

**Reply by nguyentr4n14** [reply]
Which cheat did you use in the test?

**Reply by ByMynix** [reply]
> Which cheat did you use in the test?

In the test I used GOESP. The test was only about the trust factor, so a cheat with many functions would only fake the result, as it would then come from the cheat and my possibly higher conspicuousness play. So the lower trust would come from that instead of the VAC-ByPass.
